### PR TITLE
Add JWT token import, handle viper config read error

### PR DESCRIPTION
**Pull request by lmoe**
# Description of change

Adds a simple import function for JWT keys that are still remaining inside the config. 

Additionally does some error handling when reading the config file to notify the user of structural issues with an existing config file.

### CODE REVIEW

File: tools/wasp-cli/authentication/import.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+		Short: "Imports all JWT tokens from the config into the OS Keychain",
+		Run: func(cmd *cobra.Command, args []string) {
+			tokens := config.GetAuthTokenForImport()
+
+			fmt.Println("Importing JWT tokens from the config into the OS Keychain.")
+
+			kc := config.GetKeyChain()
+			for k, v := range tokens {
+				if v == "" {
+					fmt.Printf("Could not import JWT token for node '%v'\n", k)
```
Comment (dessaya) [inline tools/wasp-cli/authentication/import.go]: Tip: use `%q` instead of `'%v'` (handles non-printable characters better)

File: tools/wasp-cli/cli/keychain/keychain.go [hunk 2]
```diff
@@ -39,7 +39,7 @@ type KeyChain interface {
 }
 
 func jwtTokenKey(node string) string {
-	return fmt.Sprintf("%s.%s", jwtTokenKeyPrefix, node)
+	return fmt.Sprintf("%s.%v", jwtTokenKeyPrefix, node)
```
Comment (dessaya) [inline tools/wasp-cli/cli/keychain/keychain.go:42]: Just curious: why this change? AFAIK `%v` defaults to `%s` for strings :thinking:

**Review by dessaya — APPROVED** [review]

**Review by jorgemmsilva — APPROVED** [review]

File: tools/wasp-cli/cli/keychain/keychain.go (hunk 2 above)
Comment (lmoe) [reply tools/wasp-cli/cli/keychain/keychain.go:42]: Just turned out to be my default as %v can also directly print other types more or less dynamically. While with %s you would get a runtime error when the variable type is different. Taken out of this context, sure a %s would fit "better" in this case. :D
